mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+"""
+mbed SDK
+Copyright (c) 2011-2015 ARM Limited
```

Comment:
year should be 2016 in some files in this PR, and we might consider updating the text as this one comes from mbed SDK (line 3)